### DISCUSSION TITLE
Webhook for readthedocs

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21703] [New to GitHub]
<p>I’m starting to use ReadTheDocs for the first time and trying to connect it to my GitHub repo. When I go into the repo on Github, select Settings then select Webhooks from the menu I thought it was supposed to display a dropdown list of webhooks, including ReadThe Docs. Instead there is no list, just an Add Webhook button which takes me to a page to set up a custom webhook.</p>
<p>Where am I going wrong?</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>There isn’t a dropdown, no. I did find <a href="https://docs.readthedocs.io/en/stable/webhooks.html#github" rel="nofollow noopener">this documentation</a> from the ReadTheDocs website that describes the process of creating a webhook for ReadTheDocs on GitHub.</p>
<p>I hope that helps!</p>
